airflow/auth/managers/simple/services/simple_auth_manager_login.py

```diff
@@ -0,0 +1,74 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
I think we can keep the filename `login.py`. Or if you really want to rename it `simple_auth_manager_login`, please rename the test filename as well

Reply:
It is a bit long name let's keep login.py. Sending in a moment. My bad!

Reply:
Adjusted the name of the file and removed additional object creation in the test since the only method is `classmethod` now.